tests/end2end/features/misc.feature

```diff
@@ -330,20 +330,17 @@ Feature: Various utility commands.
     Scenario: DNT header
         When I set content.headers.do_not_track to true
         And I open headers
-        Then the header Dnt should be set to 1
-        And the header X-Do-Not-Track should be set to 1
+        Then the header DNT should be set to 1
```

Comment:
The `Dnt` casing was correct (the page displaying the header normalizes them, as HTTP headers are case-insensitive). Since it doesn't match up with the page anymore, this now fails the test.